JASP-Engine/JASP/R/ttestbayesianindependentsamples.R

```diff
@@ -483,43 +498,29 @@ TTestBayesianIndependentSamples <- function(dataset=NULL, options, perform="run"
 						
 						plot <- plots.ttest[[z]]
 						
-						if (status[statusInd] != "error") {
-							
-							p <- try(silent= FALSE, expr= {
-							  
-							  .plotFunc <- function() {
-							    .plotPosterior.summarystats.ttest(t = tValue[i], n1 = n_group2[i], n2 = n_group1[i],
-							                                      paired = FALSE, oneSided = oneSided, BF = BF10post[i],
-							                                      BFH1H0 = BFH1H0, rscale = options$priorWidth,
-							                                      addInformation = options$plotPriorAndPosteriorAdditionalInfo,
-							                                      options = options)
-							  }
-									content <- .writeImage(width = 530, height = 400, plot = .plotFunc, obj = TRUE)
-
-									plot[["convertible"]] <- TRUE
-									plot[["obj"]] <- content[["obj"]]
-									plot[["data"]] <- content[["png"]]
-									
-								})
-							
-							if (class(p) == "try-error") {
-								
-								errorMessage <- .extractErrorMessage(p)
-								
-								if (errorMessage == "not enough data") {
-									
-									errorMessage <- "Plotting is not possible: The Bayes factor is too small"
-								} else if (errorMessage == "'from' cannot be NA, NaN or infinite") {
-									
-									errorMessage <- "Plotting is not possible: The Bayes factor is too small"
-								}
-								
-								plot[["error"]] <- list(error="badData", errorMessage=errorMessage)
-							}
-							
+						if (!is.null(errorMessage)) {
+						    
+						    plot[["error"]] <- list(error="badData", errorMessage=errorMessage)
+						    
 						} else {
 							
-							plot[["error"]] <- list(error="badData", errorMessage=plottingError[statusInd])
+						    p <- try(silent= FALSE, expr= {
```

Comment:
We will still want to catch this try. Otherwise a panel might be empty if an unexpected error occurs.